src/app/store/reducers/VehicleTestResultModel.reducers.ts

```diff
@@ -13,15 +13,38 @@ export function VehicleTestResultModelReducers(
       return {
         ...state,
         vehicleTestResultModel: action.payload,
-        error: null // clear error message
+        error: null
       };
     }
 
     case EVehicleTestResultModelActions.GetVehicleTestResultModelFailure: {
       return {
         ...state,
         vehicleTestResultModel: null,
-        error: action.payload // capture error message
+        error: action.payload
```

Comment:
also remove from here. You can dispatch setErrorMessage separately from the Effect

Reply:
setErrorMessage should not be dispatched for testResults as we are not supposed to display an error message in this case.